common/internal_model/src/test/scala/weco/catalogue/internal_model/matchers/EventuallyInElasticsearch.scala

```diff
@@ -0,0 +1,87 @@
+package weco.catalogue.internal_model.matchers
+
+import com.sksamuel.elastic4s.ElasticDsl._
+import com.sksamuel.elastic4s.requests.get.GetResponse
+import com.sksamuel.elastic4s.requests.searches.SearchResponse
+import com.sksamuel.elastic4s.{ElasticClient, Index, Response}
+import io.circe.Encoder
+import org.scalatest.concurrent.{Eventually, IntegrationPatience, ScalaFutures}
+import org.scalatest.Assertion
+import weco.elasticsearch.model.IndexId
+import weco.json.JsonUtil.toJson
+import org.scalatest.matchers.should.Matchers
+import weco.catalogue.internal_model.image.{Image, ImageState}
+import weco.catalogue.internal_model.work.{Work, WorkState}
+
+import scala.concurrent.ExecutionContext.Implicits.global
+
+//TODO: Consider turning this into a matcher?
```

Comment:
Would definitely be nice, we haven't leaned much on matchers but they're super handy.